samples/drivers/video/capture/src/main.c

```diff
@@ -144,6 +144,20 @@ int main(void)
 	}
 #endif
 
+        fmt.width = 160;
+	fmt.height = 120;
+	fmt.pitch = fmt.width * 2;
+	fmt.pixelformat = VIDEO_PIX_FMT_RGB565;
+
+	if(video_set_format(video_dev, VIDEO_EP_OUT, &fmt)){
+                LOG_ERR("Unable to set format");
+                return 0;
+        }
+
+        printk("- Set format: %c%c%c%c %ux%u\n", (char)fmt.pixelformat,
+	       (char)(fmt.pixelformat >> 8), (char)(fmt.pixelformat >> 16),
+	       (char)(fmt.pixelformat >> 24), fmt.width, fmt.height);
+
```

Comment:
What is this change?

Reply:
This is just a workaround to be able to fit the captured frame into internal ram. 
It should and will be removed once #76982 is merged.

Reply:
Ah I see. But since this impacts (negatively) users using other drivers, I would not consider this workaround acceptable. If you wanted, I guess you could add Kconfigs to the sample to allow setting a max resolution though, and based on the supported resolutions you get from the call to video_get_capabilities you could set to whatever resolutions is the closest?